gradio/components/image_editor.py

```diff
@@ -117,6 +117,25 @@ def __post_init__(self):
             )
 
 
+@document()
+@dataclasses.dataclass
+class LayerOptions:
```

Comment:
We should expose `LayersOptions` as a high-level class, so we can do `gr.LayersOptions()`